### PR TITLE
Add Crypto Papers

### PR DESCRIPTION
**Pull request by pFarb**
**[Insert URL to the list here]**

https://github.com/pFarb/awesome-crypto-papers

**[Explain what this list is about and why it should be included here]**

I maintain a list of cryptography papers for normal human beings who have to implement, use and rely on cryptography in their work and would like to understand things better without getting second PhD. Crypto theory is huge chunk of reading, and part of my job is focusing fellow engineers on reading important practical things to give them theoretical and real-world context in their work. I believe this work will be useful for many other engineers, having to implement or use encryption and trying to understand better which things they should do and which they shouldn't.

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by sindresorhus** [comment]
- Header: `Awesome crypto papers` => `Awesome Crypto Papers`
- Awesome badge should be on the right side of the title.
- Would be nice if each sub-section in the "Specifics topics" section came with a succinct description of the topic.
- You should list what license you're using. Either at the bottom of the readme or in the license file. Right now I guessed it's one of the CC licenses, but hard to tell.

---

> The image should be high-DPI. Set it to maximum half the width of the original image.

---

> The link and description are separated by a dash. 
Example: - [AVA](…) - JavaScript test runner.
The description starts with an uppercase character and ends with a period.

---

File: readme.md [hunk 1]
```diff
@@ -267,6 +267,7 @@
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby)
 - [Linguistics](https://github.com/theimpossibleastronaut/awesome-linguistics)
 - [Cryptography](https://github.com/sobolevn/awesome-cryptography)
+- [Crypto Papers](https://github.com/pFarb/awesome-crypto-papers) - focusing on theoretical background (papers, books) for using cryptographic algorithms for non-cryptographers.
```
Comment (sindresorhus) [inline readme.md]: This should be a sub-item of the `Cryptography` entry (See other examples for that above).

`focusing` => `Focusing`

And the description needs to be shorter.

**Comment by pFarb** [comment]
Updated.

File: readme.md [hunk 2]
```diff
@@ -267,6 +267,7 @@
 	- [NLP with Ruby](https://github.com/arbox/nlp-with-ruby)
 - [Linguistics](https://github.com/theimpossibleastronaut/awesome-linguistics)
 - [Cryptography](https://github.com/sobolevn/awesome-cryptography)
+	- [Papers](https://github.com/pFarb/awesome-crypto-papers) - Focusing on theoretical background for using cryptographic algorithms for non-cryptographers.
```
Comment (sindresorhus) [inline readme.md]: Can you shorten the description?

Comment (pFarb) [reply readme.md]: done

**Comment by sindresorhus** [comment]
@pFarb There are still many descriptions that don't start with an uppercase or ends with a dot.

---

Linkify `cryptography` in `A curated list of cryptography papers` to https://en.wikipedia.org/wiki/Cryptography

**Comment by pFarb** [comment]
Fixed both.

**Comment by sindresorhus** [comment]
> There are still many descriptions that don't start with an uppercase or ends with a dot.

You still need to do the ones in the `Contents` section: https://github.com/pFarb/awesome-crypto-papers#contents

**Comment by sindresorhus** [comment]
Tweet: https://twitter.com/awesome__re/status/1030568173534294016

**Comment by pFarb** [comment]
Thank you, fixed!

**Pull request merged**